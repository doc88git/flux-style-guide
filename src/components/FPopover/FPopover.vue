<template>
  <div class="f-popover">
    <f-tooltip position="top" label="Button">
      <template v-slot:content>
        tooltip content
      </template>
    </f-tooltip>
  </div>
</template>

<script>
import { FTooltip } from "../FTooltip";

export default {
  name: "f-popover",
  components: {
    FTooltip
  },
  data: () => ({
    isVisible: false
  }),
  props: {},
  computed: {},
  methods: {}
};
</script>

<style lang="scss" scoped>
.f-popover {
}
</style>
