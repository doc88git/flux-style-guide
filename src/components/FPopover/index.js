import FPopover from './FPopover'

export { FPopover }
