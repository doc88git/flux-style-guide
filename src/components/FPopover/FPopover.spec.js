import { shallowMount } from '@vue/test-utils'
import FPopover from './FPopover'

const WRAPPER = shallowMount(FPopover)

describe('Component tests', () => {
  test('Componente existe', () => {
    expect(WRAPPER.exists()).toBe(true)
  })

  test('Componente nÃ£o alterado', () => {
    expect(WRAPPER.html()).toMatchSnapshot()
  })
})
