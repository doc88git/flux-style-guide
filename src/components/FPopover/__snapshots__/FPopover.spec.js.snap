// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component tests Componente n√£o alterado 1`] = `
"<div class=\\"f-popover\\">
  <f-tooltip-stub label=\\"Button\\" position=\\"top\\" aligned=\\"center\\" size=\\"default\\" color=\\"default\\" bgcolor=\\"black\\" textcolor=\\"white\\" showevent=\\"mouseover\\" hideevent=\\"mouseleave\\" class=\\"teste\\"></f-tooltip-stub>
</div>"
`;
